tests/MoBi.Tests/Core/Commands/AddParameterToContainerCommandSpecs.cs

```diff
@@ -62,6 +62,19 @@ protected override ParameterBuildMode GetBuildModeType()
       }
    }
 
+   public class When_adding_a_new_parameter_to_interaction_container : When_testing_for_appropriate_parameter_build_modes
+   {
+      protected override IContainer GetContainer()
+      {
+         return new InteractionContainer();
+      }
+
+      protected override ParameterBuildMode GetBuildModeType()
+      {
+         return ParameterBuildMode.Global;
```

Comment:
the test

Reply:
Observations from the base test class are automatically executed when running the tests in the derived class?
I did not know it

Reply:
yeah it's because it's derived and we try to find an Obervation in each class. Pretty neat (but potentially way over engineered in this case)

Reply:
I think it's pretty usefull. 
In the case like this I have always defined a function `testXY()` in the base class and the observation in each derived class which just called `testXY()` - which could be avoided with this feature :)